### PR TITLE
Add instructions for Microsoft services codes

### PR DESCRIPTION
**Pull request by mathjazz**
Microsoft services locale codes are [now](https://github.com/mozilla/pontoon/commit/1b3fe49e920de48c3610bcf711d931fc98d5dac7) stored in the DB.

### CODE REVIEW

File: tools/pontoon/adding_new_locale.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ You will need to complete the following fields in the next page.
 
 It’s the locale code, in this case `am`.
 
+### MS Translator Code
```
Comment (flodolo) [inline tools/pontoon/adding_new_locale.md]: MS translator code (lowercase)

File: tools/pontoon/adding_new_locale.md [hunk 2]
```diff
@@ -14,6 +14,14 @@ You will need to complete the following fields in the next page.
...
+
+Microsoft Translator maintains a list of supported locales in its own format. Choose one that matches the locale from [a list of supported locales](https://msdn.microsoft.com/en-us/library/hh456380.aspx) or leave it blank to disable support for Microsoft Translator for this locale.
+
+### MS Terminology Code
```
Comment (flodolo) [inline tools/pontoon/adding_new_locale.md]: MS terminology code

File: tools/pontoon/adding_new_locale.md [hunk 3]
```diff
@@ -14,6 +14,14 @@ You will need to complete the following fields in the next page.
...
+
+Microsoft Terminology maintains a list of supported locales in its own format. Choose one that matches the locale from a list provided in the admin below the field or leave it blank to disable support for Microsoft Terminology for this locale. The
```
Comment (flodolo) [inline tools/pontoon/adding_new_locale.md]: I would remove "in the admin".

There's an extra "The" at the end.

**Review by flodolo — CHANGES_REQUESTED** [review]

**Comment by mathjazz** [comment]
Updated.

**Review by flodolo — APPROVED** [review]

**Pull request merged**